### PR TITLE
MTDSA-16861: OAS Update for Property Business API (sandbox changes)

### PR DESCRIPTION
**Pull request by LouiseWong98**

### CODE REVIEW

**Review by rhysconnor — APPROVED** [review]
LGTM - waiting to merge

File: resources/public/api/conf/2.0/scenarios/default_not_found_stateful_scenarios.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        </tr>
+    </tbody>
+    <tbody>
+        <tr>
+            <td><p>NOT_FOUND</p></td>
+            <td><p>Simulates the scenario where no data is found.</p></td>
+        </tr>
+          <tr>
+              <td><p>STATEFUL</p></td>
+              <td><p>Simulates a stateful success response.</p></td>
```
Comment (rhysconnor) [inline resources/public/api/conf/2.0/scenarios/default_not_found_stateful_scenarios.md:22]: Do we need to update the wording for this STATEFUL scenario?

File: resources/public/api/conf/2.0/scenarios/default_not_found_stateful_scenarios.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
```
Comment (ben-HMRC) [inline resources/public/api/conf/2.0/scenarios/default_not_found_stateful_scenarios.md:1]: "is only available" to "are only available"

**Review by ben-HMRC — DISMISSED** [review]

**Review by rhysconnor — APPROVED** [review]
LGTM

**Pull request merged**